patroni/dcs/__init__.py

```diff
@@ -1156,106 +1570,158 @@ def manual_failover(self, leader: Optional[str], candidate: Optional[str],
         """Race for cluster initialization.
 
-        :param create_new: False if the key should already exist (in the case we are setting the system_id)
-        :param sysid: PostgreSQL cluster system identifier, if specified, is written to the key
-        :returns: `!True` if key has been created successfully.
+        This method should atomically create ``initialize`` key and return ``True``,
+        otherwise it should return ``False``.
+
+        :param create_new: ``False`` if the key should already exist (in the case we are setting the system_id).
+        :param sysid: PostgreSQL cluster system identifier, if specified, is written to the key.
 
-        this method should create atomically initialize key and return `!True`
-        otherwise it should return `!False`"""
+        :returns: ``True`` if key has been created successfully.
+        """
 
     @abc.abstractmethod
     def _delete_leader(self) -> bool:
         """Remove leader key from DCS.
-        This method should remove leader key if current instance is the leader"""
+
+        This method should remove leader key if current instance is the leader.
+
+        :returns: ``True`` if successfully committed to DCS.
+        """
 
     def delete_leader(self, last_lsn: Optional[int] = None) -> bool:
-        """Update optime/leader and voluntarily remove leader key from DCS.
+        """Update ``optime/leader`` and voluntarily remove leader key from DCS.
+
         This method should remove leader key if current instance is the leader.
-        :param last_lsn: latest checkpoint location in bytes"""
 
+        :param last_lsn: latest checkpoint location in bytes.
+
+        :returns: boolean result of called abstract :meth:`~AbstractDCS._delete_leader`.
+        """
         if last_lsn:
             self.write_status({self._OPTIME: last_lsn})
         return self._delete_leader()
 
     @abc.abstractmethod
     def cancel_initialization(self) -> bool:
-        """ Removes the initialize key for a cluster """
+        """Removes the ``initialize`` key for a cluster.
+
+        :returns: ``True`` if successfully committed to DCS.
+        """
 
     @abc.abstractmethod
     def delete_cluster(self) -> bool:
-        """Delete cluster from DCS"""
+        """Delete cluster from DCS.
+
+        :returns: ``True`` if successfully committed to DCS.
+        """
 
     @staticmethod
     def sync_state(leader: Optional[str], sync_standby: Optional[Collection[str]]) -> Dict[str, Any]:
-        """Build sync_state dict.
-        The sync_standby key being kept for backward compatibility.
-        :param leader: name of the leader node that manages /sync key
-        :param sync_standby: collection of currently known synchronous standby node names
-        :returns: dictionary that later could be serialized to JSON or saved directly to DCS
+        """Build ``sync_state`` dictionary.
+
+        The *sync_standby* key is kept for backward compatibility.
```

Comment:
I think we wanted to say that we don't use plural (`sync_standbys`) for backward compatibility.
```suggestion
```